ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-master.yaml

```diff
@@ -5,9 +5,17 @@ base_images:
     tag: base
 build_root:
   project_image:
-    context_dir: hack/kubevirt-builder
-    dockerfile_path: Dockerfile
+    dockerfile_path: hack/kubevirt-builder/Dockerfile.ci
 canonical_go_repository: kubevirt.io/kubevirt
+images:
```

Comment:
So you are not using the `src` image above any more, right?

Reply:
right, still working that out.. i'm going to have to ask testplatform team to clarify this.

Reply:
The LOCAL_IMAGE_KUBEVIRT_TEST is the image built from this PR's Dockerfile: https://github.com/kubevirt/kubevirt/pull/2991  
Is there another image you were using b4?

Reply:
The custom-test-image template calls for a test-image variable, I'm using the env var 
`TEST_IMAGESTREAM_TAG=stable:kubevirt-test`